src/Microsoft.Health.CosmosDb/Configs/CosmosDataStoreConfiguration.cs

```diff
@@ -20,6 +20,14 @@ public class CosmosDataStoreConfiguration
 
         public int? InitialDatabaseThroughput { get; set; }
 
+        public string FhirCollectionId { get; set; }
```

Comment:
These properties shouldn't be added. They were refactored out to the various collection configs.